examples/Common.Tests.ps1

```diff
@@ -48,3 +48,76 @@ Describe 'Verify Resource Exists' {
     $exists | Should -Be $true
   }
 }
+
+Describe 'Use Confirm-AzBPResource to confirm resource and/or properties exist'{
+  Describe 'Verify Container Registry' {
+    it 'Should contain a container registry named testcontaineregistry' {
+      #arrange
+      $resourceGroupName = "testrg"
+      $resourceType = "ContainerRegistry"
+      $containerRegistryName = "testcontaineregistry"
+
+      #act
+      $result = Confirm-AzBPResource -ResourceGroupName $resourceGroupName -ResourceType $resourceType -ResourceName $containerRegistryName
+
+      #assert
+      $result.Success | Should -Be $true
+    }
+
+    it 'Should contain a container registry named testcontaineregistry is Standard' {
+      #arrange
+      $resourceGroupName = "testrg"
+      $resourceType = "ContainerRegistry"
+      $containerRegistryName = "testcontaineregistry"
+      $property = "SkuName"
+      $expectedValue = "Standard"
+
+      #act
+      $result = Confirm-AzBPResource -ResourceGroupName $resourceGroupName -ResourceType $resourceType -ResourceName $containerRegistryName -PropertyKey $property -PropertyValue $expectedValue
```

Comment:
This line length looks long, is it < 120?

Reply:
Hm strange megalinter did not pick this up. I am assuming (can't help myself to assume lol) that it defaults to ignore test files...anyways, resolved the line length.